docs/tutorials/dataflows/nlp.rst

```diff
@@ -0,0 +1,89 @@
+Using NLP Operations
+==============================
+
+This example will show you how to use DFFML operations to clean text data and train a model using DFFML cli.
+
+DFFML offers several :ref:`plugin_models`. For this example
+we will be using the tensorflow DNNClassifier model
+(:ref:`plugin_model_dffml_model_tensorflow_tfdnnc`) which is in the ``dffml`` package.
+
+We will use two operations :ref:`plugin_operation_dffml_operations_nlp_remove_stopwords` and :ref:`plugin_operation_dffml_operations_nlp_get_embedding`.
+Internally, both of these operations use `spacy <https://spacy.io/usage/spacy-101>`_ functions.
```

Comment:
We usually ask the user to install the plugins we'll be using after we explain what pieces in those plugins we'll be using

```

.. code-block:: console

    $ pip install -U dffml-model-tensorflow dffml-operations-nlp

```